tests/ClassicalField/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_executable(Laplace Laplace.f90)
+set_target_properties(Laplace PROPERTIES LINKER_LANGUAGE Fortran)
```

Comment:
Yes, remember I mentioned it to you. For some reason it isn't selecting the Fortran compiler for the link and is using the C one. It then has unresolved symbols as the C compiler doesn't link in the Fortran run time libraries. I'm not sure why it is not selecting Fortran from the .f90 source file but I needed to add in the linker language properties to get it to work.